### DISCUSSION TITLE
OpenMMLab 2.0公测 中文讨论区

### DISCUSSION THREAD

**Discussion by vansin** [open-mmlab/mmaction2#1929] [General]
OpenMMLab 2.0公测 中文讨论区

**Comment by Zheng-LinXiao** [comment]
在1.x的版本中，tsn的results and models部分中kinetics-400的ckpt重复。（链接：https://github.com/open-mmlab/mmaction2/tree/1.x/configs/recognition/tsn  ）

**Comment by Zheng-LinXiao** [comment]
在mmaction2 1.x中get_started.md部分中（链接：https://github.com/open-mmlab/mmaction2/blob/1.x/docs/en/get_started.md ）， verify the installation的步骤1，使用mim下载config和checkpoint会报错（文档中的命令：mim download mmaction2 --config tsn_r50_8xb32-1x1x8-100e_kinetics400-rgb --dest .），显示文件问题。
然后我在1.x和dev-1.x中利用tsn对比发现，
在1.x的tsn-kinetics-400（https://github.com/open-mmlab/mmaction2/tree/1.x/configs/recognition/tsn ）中，它四个所对应的ckpt地址不对（四个对应的位置都是同一个）。
在dev-1.x的tsn-kinetics-400（https://github.com/open-mmlab/mmaction2/tree/dev-1.x/configs/recognition/tsn ）中，它的config和checkpoint对应是正确的（但是ckpt在下载时是空的），但是他们的命名格式与get_started.md（最上面描述的问题）的不一致。
我不知道mim download mmaction2 --config tsn_r50_8xb32-1x1x8-100e_kinetics400-rgb --dest . 这个命令的错误是不是这些对应的问题造成的。
我尝试过把命令中tsn_r50_8xb32-1x1x8-100e_kinetics400-rgb改成tsn_imagenet-pretrained-r50_8xb32-1x1x8-100e_kinetics400-rgb.py 也是错误的.

**Comment by Zheng-LinXiao** [comment]
我举了tsn三个分支里面的config命名情况，
master：
tsn_r50_1x1x3_100e_kinetics400_rgb
1.x：
tsn_r50_1x1x3_100e_8xb32_kinetics400_rgb
dev-1.x：
tsn-imagenet-pretrained-50_8xb32-1x1x3-100e kinetic400-rgb
在master是正常使用的，在1.x中，ckpt对应的名字是master与1.x不符。而在dev-1.x中，config与ckpt名字都是相对应的，但是在下载dev-1.x的ckpt显示是空白。所以在1.x和dev-1.x的config中我都无法验证，跑demo。

**Reply by hukkai** [reply]
1.x的ckpt还没有上传，最近会上传

**Reply by Zheng-LinXiao** [reply]
知道啦，谢谢大佬的回复
